components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/internal/FrameworkServiceComponent.java

```diff
@@ -757,4 +767,31 @@ private AuthenticatorConfig getAuthenticatorConfig(String name) {
         }
         return authConfig;
     }
+
+    @Reference(
+            name = "SessionStore.",
```

Comment:
Let's use a proper name value here. You may find a better name in the SessionStore class's service component